### PR TITLE
Adding OpenNetCDF to frmMain

### PR DESCRIPTION
**Pull request by AlexSananka**
@dannyparsons I had forgot to add dlgOpenNetCDF to the frmmain but I have done it now,please merge it for @stevekogo needs it.

### CODE REVIEW

File: instat/frmMain.Designer.vb [hunk 1]
```diff
@@ -993,44 +994,44 @@ Partial Class frmMain
...
         Me.mnuModelOtherTwoVariablesTwoSamples.Tag = "Two_Samples..."
         Me.mnuModelOtherTwoVariablesTwoSamples.Text = "Two Samples..."
         '
         'mnuModelOtherTwoVariablesSummaryData
         '
         Me.mnuModelOtherTwoVariablesSummaryData.Enabled = False
         Me.mnuModelOtherTwoVariablesSummaryData.Name = "mnuModelOtherTwoVariablesSummaryData"
-        Me.mnuModelOtherTwoVariablesSummaryData.Size = New System.Drawing.Size(266, 22)
+        Me.mnuModelOtherTwoVariablesSummaryData.Size = New System.Drawing.Size(267, 22)
         Me.mnuModelOtherTwoVariablesSummaryData.Tag = "Summary_Data"
```
Comment (Lunalo) [inline instat/frmMain.Designer.vb]: Tag should have Ellipsis

File: instat/frmMain.Designer.vb [hunk 2]
```diff
@@ -993,44 +994,44 @@ Partial Class frmMain
...
         Me.mnuModelOtherTwoVariablesSummaryData.Text = "Summary Data..."
         '
         'mnuModelOtherTwoVariablesSimpleRegression
         '
         Me.mnuModelOtherTwoVariablesSimpleRegression.Name = "mnuModelOtherTwoVariablesSimpleRegression"
-        Me.mnuModelOtherTwoVariablesSimpleRegression.Size = New System.Drawing.Size(266, 22)
+        Me.mnuModelOtherTwoVariablesSimpleRegression.Size = New System.Drawing.Size(267, 22)
         Me.mnuModelOtherTwoVariablesSimpleRegression.Tag = "Simple_Regression"
```
Comment (Lunalo) [inline instat/frmMain.Designer.vb]: Ellipsis missing

File: instat/frmMain.Designer.vb [hunk 3]
```diff
@@ -993,44 +994,44 @@ Partial Class frmMain
...
         Me.mnuModelOtherTwoVariablesSimpleRegression.Text = "Simple Regression..."
         '
         'mnuModelOtherTwoVariablesOneWayANOVA
         '
         Me.mnuModelOtherTwoVariablesOneWayANOVA.Name = "mnuModelOtherTwoVariablesOneWayANOVA"
-        Me.mnuModelOtherTwoVariablesOneWayANOVA.Size = New System.Drawing.Size(266, 22)
+        Me.mnuModelOtherTwoVariablesOneWayANOVA.Size = New System.Drawing.Size(267, 22)
         Me.mnuModelOtherTwoVariablesOneWayANOVA.Tag = "One_Way_ANOVA"
```
Comment (Lunalo) [inline instat/frmMain.Designer.vb]: Same as here

File: instat/frmMain.Designer.vb [hunk 4]
```diff
@@ -993,44 +994,44 @@ Partial Class frmMain
...
         Me.mnuModelOtherTwoVariablesOneWayANOVA.Text = "One Way ANOVA..."
         '
         'mnuModelOtherTwoVariablesNonParametricTwoSamples
         '
         Me.mnuModelOtherTwoVariablesNonParametricTwoSamples.Enabled = False
         Me.mnuModelOtherTwoVariablesNonParametricTwoSamples.Name = "mnuModelOtherTwoVariablesNonParametricTwoSamples"
-        Me.mnuModelOtherTwoVariablesNonParametricTwoSamples.Size = New System.Drawing.Size(266, 22)
+        Me.mnuModelOtherTwoVariablesNonParametricTwoSamples.Size = New System.Drawing.Size(267, 22)
         Me.mnuModelOtherTwoVariablesNonParametricTwoSamples.Tag = "Non_Parameteric_Two_Samples"
```
Comment (Lunalo) [inline instat/frmMain.Designer.vb]: Same as here

File: instat/frmMain.Designer.vb [hunk 5]
```diff
@@ -1047,21 +1048,21 @@ Partial Class frmMain
         '
         Me.mnuModelOtherThreeVariablesSimpleWithGroups.Enabled = False
         Me.mnuModelOtherThreeVariablesSimpleWithGroups.Name = "mnuModelOtherThreeVariablesSimpleWithGroups"
-        Me.mnuModelOtherThreeVariablesSimpleWithGroups.Size = New System.Drawing.Size(260, 22)
+        Me.mnuModelOtherThreeVariablesSimpleWithGroups.Size = New System.Drawing.Size(262, 22)
         Me.mnuModelOtherThreeVariablesSimpleWithGroups.Tag = "Simple_with_groups"
```
Comment (Lunalo) [inline instat/frmMain.Designer.vb]: Technical word should start with Upper Case

File: instat/frmMain.Designer.vb [hunk 6]
```diff
@@ -1047,21 +1048,21 @@ Partial Class frmMain
...
         Me.mnuModelOtherThreeVariablesNonParametricTwoWayANOVA.Tag = "Non_Parametric_Two_Way_ANOVA..."
         Me.mnuModelOtherThreeVariablesNonParametricTwoWayANOVA.Text = "Non Parametric Two Way ANOVA..."
         '
         'mnuModelOtherThreeVariablesChisquareTest
         '
         Me.mnuModelOtherThreeVariablesChisquareTest.Name = "mnuModelOtherThreeVariablesChisquareTest"
-        Me.mnuModelOtherThreeVariablesChisquareTest.Size = New System.Drawing.Size(260, 22)
+        Me.mnuModelOtherThreeVariablesChisquareTest.Size = New System.Drawing.Size(262, 22)
         Me.mnuModelOtherThreeVariablesChisquareTest.Tag = "Chi-square_Test"
         Me.mnuModelOtherThreeVariablesChisquareTest.Text = "Chi-square Test..."
```
Comment (Lunalo) [inline instat/frmMain.Designer.vb]: c and s should either be in lower or upper

File: instat/frmMain.Designer.vb [hunk 7]
```diff
@@ -1078,23 +1079,23 @@ Partial Class frmMain
...
         Me.mnuModelOtherGeneralANOVAGeneral.Tag = "ANOVA_General..."
         Me.mnuModelOtherGeneralANOVAGeneral.Text = "ANOVA General..."
         '
         'mnuModelOtherGeneralRegression
         '
         Me.mnuModelOtherGeneralRegression.Enabled = False
         Me.mnuModelOtherGeneralRegression.Name = "mnuModelOtherGeneralRegression"
-        Me.mnuModelOtherGeneralRegression.Size = New System.Drawing.Size(166, 22)
+        Me.mnuModelOtherGeneralRegression.Size = New System.Drawing.Size(167, 22)
         Me.mnuModelOtherGeneralRegression.Tag = "Regression"
```
Comment (Lunalo) [inline instat/frmMain.Designer.vb]: Just take not of Ellipsis

File: instat/frmMain.Designer.vb (hunk 5 above)
Comment (AlexSananka) [inline instat/frmMain.Designer.vb]: A dash implies that's one word.

**Comment by rdstern** [comment]
I like some of the changes in frmMain.  However the **Describe** and **Model** menus have swapped over.  I am not sure how.  Please could they be returned to their previous locations.  As a main menu, **Describe is before Model**.

**Pull request merged**